packages/miniflare/src/plugins/core/index.ts

```diff
@@ -152,6 +155,7 @@ const CoreOptionsSchemaInput = z.intersection(
 		 (If there are assets but we're not using vitest, the miniflare entry worker can point directly to RW.)
 		 */
 		hasAssetsAndIsVitest: z.boolean().optional(),
+		unsafeEnableAssetsRpc: z.boolean().optional(),
```

Comment:
Is this the right place for this? I'm pretty sure Assets RPC should be a single on/off switch for _all_ workers, not per-worker

Reply:
removed